index.html

```diff
@@ -39,6 +42,201 @@ <h4>LET'S CONNECT</h4>
+                            <li><img class="dot" img src="css/img/Counter.png"></li>
+                            <li class="frame3">2015</li>      
+                        </ul>
+                    </div>
+                </div>
+                <p class="primary-text"> 
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+                <div class="tags">
+                    <ul>
+                        <li><a>HTML</a></li>
+                        <li><a>CSS</a></li>
+                        <li><a>JavaScript</a></li>
+                    </ul>
+                </div>
+                <div class="action">
+                    <a>See project</a>
+                </div>
+            </div>
+
+        </div>
+        </section>
+        <section id="myself">
+            <div id="container-text-icons">
+                <div id="text-presentation">
+                  <h1>About Myself</h1>
+                    <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                    <h4>LET'S CONNECT</h4>
+                </div>
+                <div id="container-icons">
+                    <ul>
+                        <li><a><img id="twitter"src="css/img/twitter-icon.png" alt="Twitter"></a></li>
+                        <li><a><img id="link" src="css/img/Linkedin-icon.png" alt="LinkedIn"></a></li>
+                        <li><a><img id="m" src="css/img/M-icon.png" alt=""></a></li>
+                        <li><a><img id="github" src="css/img/github-icon.png" alt="GitHub"></a></li>
+                        <li><a><img id="paz" src="css/img/paz-icon.png" alt=""></a></li>
+                    </ul> 
+                </div>
+               <div id="resume">
+                <a>Get my resume</a>
+               </div> 
+            </div>
+            <div id="skills">
+                <ul>
+                    <li class="skills-li">
+                        <a>
+                            <h2>Languages</h2>
+                             <img  class="img-bottom" src="css/img/arrowbotton.png">
+                        </a>
+                    </li>
+                        <ul class="sub-list">
+                            <li>
+                                <img src="css/img/JS.png">
+                                <h3>JavaScript</h3>
+                            </li>
+                            <li>
+                                <img src="css/img/HTML.png">
+                                <h3>HTML</h3>
+                            </li>
+                            <li>
+                                <img src="css/img/css3.png">
+                                <h3>CSS</h3>
+                            </li>

```

Comment:
- [ ] Kindly adjust the following.

- Your design
<img width="238" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/63a5c7cd-8307-404a-9971-fa15006d3b07">

- Figma design
<img width="232" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/ac2bf4b8-dd6e-475b-9759-96bbacbb3f0f">

Reply:
- [ OPTIONAL] Great! now I can see that you have fixed your icon position and made it according to Figma. 😊👍 However, I want to suggest you another simple thing, as you can see in the Figma design there is a border-radius of 8px in each list item of the language. Please apply this border to your list items of the `sub-list`, so that your design will look more beautiful and match with that of Figma. 😊